e2e/fixtures/plugin/doc/components/guide.mdx

```diff
@@ -13,3 +13,10 @@ export default function HelloWorld() {
 ```
 
 <code src="./component.jsx" />
+
+```json
```

Comment:
seems no test cases about preview plugin in e2e/tests/plugin.test.ts? Where we test it?

Reply:
I split the test cases of the plugin. The preview plugin test cases are located in `e2e/fixtures/plugin-preview` and `plugin-preview.test.ts 
`

Reply:
@Timeless0911